### PR TITLE
animals table create, post and query

### PR DESCRIPTION
**Pull request by paulshaun-kdtk**
# Changes

- installed postgres
- created vet_clinic db inside postgres
- created animals table
- added info to animals table
- queried animals table

## query screenshot attachments

Screenshot (258)

Screenshot (259)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INTEGER,
```
Comment (topeogunleye) [inline schema.sql:4]: - [OPTIONAL] As an enhancement, consider making the `id` column **not null** and **autoincremented** with `id int NOT NULL IDENTITY(1,1)`.
- [OPTIONAL] Marking the `id` column as a **primary key** would be a great addition for ensuring unique records! Consider adding `PRIMARY KEY (id)`.

File: README.md [hunk 2]
```diff
@@ -98,23 +98,23 @@ After you're finished please remove all the comments and instructions!
...
 
-### Key Features <a name="key-features"></a>
+<!-- ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
 - **[key_feature_1]**
 - **[key_feature_2]**
-- **[key_feature_3]**
+- **[key_feature_3]** -->
```
Comment (topeogunleye) [inline README.md:107]: - [OPTIONAL] Please kindly add the future features of this application. 
Edit: I meant key features

**Review by topeogunleye — APPROVED** [review]
Hi @paulshaun-kdtk, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

dwayne-the-rock

## (Highlights) Good Points: 💪 

Hello @paulshaun-kdtk! 🌟

I'm truly impressed with your SQL project submission! Let's go over your incredible work step by step:

### Repository and File Structure: 📁
- You've accurately set up your repo using the specified template. The presence of `schema.sql`, `data.sql`, and `queries.sql` ensures that you've effectively organized your SQL code into logical segments. Well done! 👏

### `schema.sql` Highlights: 🛠️
- Your `animals` table structure aligns perfectly with the requirements. You've correctly defined the data types, and the layout is clean and intuitive. 🐾

### `data.sql` Highlights: 📚
- Spot on with the data insertion! Your `INSERT` statements for the animals `'Agumon', 'Gabumon', 'Pikachu', and 'Devimon'` meet the project's instructions precisely. ✅

### `queries.sql` Insights: 🧐
- Your query to fetch animals with names ending in "mon" is spot-on. 🎯
- Impressive flexibility in your queries! You've showcased multiple ways to filter results, like using both raw date comparisons and the `EXTRACT` function for year extraction.
- All your queries, from filtering neutered animals to weighing animals, are on point. Your understanding and application of SQL functions and operators is commendable! 🚀

### Screenshots and Validation: 🖼️
- I appreciate the screenshots added to the pull request. They not only validate your queries but also affirm that they were successfully executed and produced the expected results. Always a great practice! 📸

### Overall: 💯
Your submission not only fulfills but excels in meeting the project requirements! Your dedication to detail, proper code organization, and deep understanding of SQL shines through in this project.

Keep up this amazing momentum, and never stop exploring the depths of SQL! 🚀🔍

If you have any further questions or wish to delve deeper into any aspect, don't hesitate to reach out.

Warm regards and happy querying! 🎉👩‍💻

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- For future pull requests, please notice that your branch name `vet_clinic` does not follow the convention of being named after a feature or action. For example, you could name your branch `added_vet_clinic`. This makes it easier to track the changes and the purpose of each branch.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (paulshaun-kdtk) [reply README.md:107]: hi @topeogunleye thank you so much for the kind review.. i will definitely add the features you suggessted.

Comment (topeogunleye) [reply README.md:107]: You're welcome @paulshaun-kdtk.

**Pull request merged**